include/arch/x86/linker.ld

```diff
@@ -85,6 +85,7 @@ SECTIONS
 #else
 	_image_text_start = .;
 #endif
+
```

Comment:
I don't understand why this is happening in the x86 linker script.  Can't you put it in common-rom.ld so all architectures can use it?